src/components/Cewl/Cewl.tsx

```diff
@@ -29,22 +19,50 @@ interface FormValuesType {
-    const [checkedLowercase, setCheckedLowercase] = useState(false);
-    const [authType, setAuthType] = useState("");
-    const [allowSave, setAllowSave] = useState(false);
-    const [hasSaved, setHasSaved] = useState(false);
+    // Component State Variables.
+    const [loading, setLoading] = useState(false); // State variable to indicate loading state.
+    const [output, setOutput] = useState(""); // State variable to store the output of the command execution.
+    const [pid, setPid] = useState(""); //  State variable to store the process ID of the command execution.
+    const [allowSave, setAllowSave] = useState(false); //   State variable to allow saving the output to a file.
+    const [hasSaved, setHasSaved] = useState(false); // State variable to indicate if the output has been saved.
+    const [loadingModal, setLoadingModal] = useState(true); // State variable to indicate loading state of the modal.
+    const [isCommandAvailable, setIsCommandAvailable] = useState(false); // State variable to check if the command is available.
+    const [opened, setOpened] = useState(!isCommandAvailable); // State variable that indicates if the modal is opened.
+    const [authType, setAuthType] = useState(""); // State variable that indicates authentication.
+    const [checkedAdvanced, setCheckedAdvanced] = useState(false); // State variable to toggle advanced mode.
+    const [checkedNumber, setCheckedNumber] = useState(false); // State variable to include words with numbers.
+    const [checkedCount, setCheckedCount] = useState(false); // State variable to show the count for each word found.
+    const [checkedEmail, setCheckedEmail] = useState(false); // State variable to include and show email addresses.
+    const [checkedLowercase, setCheckedLowercase] = useState(false); // State variable to show only lowercase words.
 
+    // Component Constants.
+    const methods = ["No authentication", "Basic", "Digest"]; // Variable to store authentication methods.
+    const title = "Cewl"; // Title of the component.
+    const description =
+        "The tool Cewl, renown for being a custom word list generator, is a ruby app which spiders given URLs to " +
+        "a specified depth to return a list of words that are able to be used within password crackers including " +
+        " JohnTheRipper (which can be found within the DDT). This tool is particularly useful for security testing and " +
+        "forensic investigation.\nOptions for the tool can be found at:  https://www.kali.org/tools/cewl/#:~:text=CeWL" +
+        "%20(Custom%20Word%20List%20generator,\nCeWL%20can%20follow%20external%20links."; // Description of the component.
+    const steps =
+        "Step 1: Enter the maximum depth to spider to. Eg: 2\n" +
+        "Step 2: Enter a minimum word length. Eg: 3\n" +
+        "Step 3: Enter a target URL. Eg: google.com\n" +
+        "Step 4: Click 'Scan' to commence Cewl's operation.\n" +
```

Comment:
Change scan to Start <title> 
```suggestion
        "Step 4: Click 'Start " + title + "' to commence Cewl's operation.\n" +
```

Reply:
done

Reply:
I think you might have missed this one (?) 🤔